src/accelerate/accelerator.py

```diff
@@ -1936,7 +1936,7 @@ def get_tracker(self, name: str, unwrap: bool = False):
         if len(getattr(self, "trackers", [])) > 0:
             for tracker in self.trackers:
                 if tracker.name == name:
-                    return tracker
+                    return tracker.tracker if unwrap else tracker
```

Comment:
Can we also update the docstring to state what it will return in this case? (The full tracker if `True` else the `Tracker`